Gemfile.lock

```diff
@@ -107,7 +107,7 @@ GEM
     govuk_frontend_toolkit (7.2.0)
       railties (>= 3.1.0)
       sass (>= 3.2.0)
-    govuk_publishing_components (21.22.1)
+    govuk_publishing_components (22.0.0)
```

Comment:
There is no version 22.
21.22.1 is latest

Reply:
There is on rubygems.org which we yanked... https://rubygems.org/gems/govuk_publishing_components/versions/22.0.0

Reply:
Have pinned to 21 for now @alex-ju

Reply:
I thought the yanked version is gone for good. Now I'm questioning what will happen when we'll try and release 22.0.0. I'll try and dig a bit into this as it can happen fairly soon.